### DISCUSSION TITLE
blog/multitenancy-in-aspnet-core/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#36] [General]
# blog/multitenancy-in-aspnet-core/

In this article, let's learn how to implement Multitenancy in ASP.NET Core in a rather simple way making use of Entity Framework Core.

https://codewithmukesh.com/blog/multitenancy-in-aspnet-core/

**Comment by timsar2** [comment]
Greate mukesh, How can I add entity to table if there is no api call with token? Like when we need to add some entity periodicaly and we just know the tenant id and we have no token in context
